### PR TITLE
Updated React post for new dev console

### PR DESCRIPTION
**Pull request by nbarbettini**
New screenshot and text to bring this blog post in line with the new dev console experience.

### CODE REVIEW

**Comment by nbarbettini** [comment]
Since I can't add a reviewer myself, ping @leebrandt for review.

**Comment by mraible** [comment]
Lowercase extension: `_source/_assets/img/blog/react-sign-in-widget/react-sample-app-settings.PNG` to `_source/_assets/img/blog/react-sign-in-widget/react-sample-app-settings.png`

File: _source/_posts/2017-03-30-react-okta-sign-in-widget.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ When you open `http://localhost:3000`, you should see something like this:
 
 When you click on the navigation links, you should see page placeholders for those links.
 ## Add the Okta Sign-In Widget
-Install the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) using NPM. We’ll be using version 1.9.0 of the Sign-In Widget, which is the most recent version at the time of this writing. Note that [using Yarn won't work](https://github.com/okta/okta-signin-widget/issues/191).
+Install the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) using NPM. We’ll be using version 2.1.0 of the Sign-In Widget, which is the most recent version at the time of this writing.
```
Comment (mraible) [inline _source/_posts/2017-03-30-react-okta-sign-in-widget.md]: NPM > npm

File: _source/_posts/2017-03-30-react-okta-sign-in-widget.md [hunk 2]
```diff
@@ -73,15 +73,14 @@ and then add the route inside the main route (the one with the path of "/")
...
-In order to *use* Okta as your OpenID Connect provider for authentication, you’ll need to set up an application in the [Okta developer portal](https://developer.okta.com).
+In order to use Okta as your OpenID Connect provider for authentication, you’ll need to set up an application in the [Okta developer console](https://developer.okta.com).
 
-So log in to your Okta account, [or create one](https://developer.okta.com) if you haven’t yet. Navigate to Admin > Add Applications and click on the Create New App button. Select Single Page App (SPA) for Platform and OpenID Connect for the sign on method. Click the Create button and give your application a name. On the next screen, add `http://localhost:3000` as a Redirect URI and click Finish. You should see settings like the following.
+If you don't have an Okta developer account, [go create one](https://developer.okta.com/signup/)! Once you're logged into the developer console, click on Applications in the top navbar, then click Add Application. Select SPA as the platform and click Next. Change the redirect URI to `http://localhost:3000`, and click Done. The application will be created with the following settings:
```
Comment (mraible) [inline _source/_posts/2017-03-30-react-okta-sign-in-widget.md]: I'd drop "developer console" and bold the button clicks - **Applications** and **Add Application**

File: _source/_posts/2017-03-30-react-okta-sign-in-widget.md (hunk 1 above)
Comment (nbarbettini) [reply _source/_posts/2017-03-30-react-okta-sign-in-widget.md]: D'oh

**Comment by nbarbettini** [comment]
Thanks for the review @mraible! Updated and rebased.

**Comment by mraible** [comment]
@nbarbettini I'd recommend adding a changelog at the end too. Like I did in https://github.com/okta/okta.github.io/commit/a6081e2bb8b31b9d2764a13452442f5394538eb2.

**Comment by nbarbettini** [comment]
Good thought, added the changelog.

@leebrandt I just remembered that your sample application will need to be updated to use the widget v2.1.0.

**Comment by mraible** [comment]
@nbarbettini Let's wait until the sample application is updated before merging this. Then we can link to the diff in the changelog.

**Comment by leebrandt** [comment]
The github project uses v 2.1.0. Should be G2G

**Pull request merged**